AUTHORS

```diff
@@ -0,0 +1,11 @@
+# This the official list of authors for copyright purposes.
```

Comment:
As per advice from our open source programs office, this text should read
```
# This is the list of bazelbuild/platforms' significant contributors.
#
# This does not necessarily list everyone who has contributed code,
# especially since many employees of one corporation may be contributing.
# To see the full list of contributors, see the revision history in
# source control.
Google LLC
```

and then the other names.

Reply:
Done.